driver/conv_driver.hpp

```diff
@@ -3153,7 +3153,7 @@ int ConvDriver<Tgpu, Tref>::RunBackwardDataGPUReference()
         din_dev->FromGPU(GetStream(), din_host.data.data());
     else
     {
-        auto din_tmp = tensor<Tgpu>(miopen::deref(inputTensor));
+        auto din_tmp = tensor<Tgpu>(miopen::ToApi(miopen::deref(inputTensor).GetType()));
```

Comment:
`ToApi` should not be here. It should be used only when we want to call public API function, e.g. `miopenSetTensorDescriptor`.

When you remove `ToApi`, you will see a build error. The reason of the error is incorrect `.GetType()` which should be removed.